.travis.yml

```diff
@@ -91,160 +53,150 @@ matrix:
+               TOXPOSARGS="--remote-data=astropy"
+               CONDA_DEPENDENCIES="clang"
                CCOMPILER=clang
+               USE_CI_HELPERS=True
                EVENT_TYPE='cron'
-               DOUBLE_PLAY=True
+               # DOUBLE_PLAY=True
 
         # Try all python versions and Numpy versions. Since we can assume that
         # the Numpy developers have taken care of testing Numpy with different
         # versions of Python, we can vary Python and Numpy versions at the same
         # time.
-        - os: linux
-          env: PYTHON_VERSION=3.6 NUMPY_VERSION=1.16
-               INSTALL_CMD='python setup.py build_ext --inplace'
-               PIP_DEPENDENCIES='pytest-astropy'
-               TEST_CMD='pytest --open-files --doctest-rst'
-          script:
-            - $INSTALL_CMD
-            - $TEST_CMD
-
-        # Now try with all optional dependencies. We also include the --readonly
-        # flag to make sure no files are being written to the temporary install
-        # location during testing. We also use this build to make sure that the
-        # dependencies get correctly installed with pip.
-        - os: linux
-          env: SETUP_CMD='test --remote-data=astropy --readonly'
-               LC_CTYPE=C.ascii LC_ALL=C
-               PIP_DEPENDENCIES="" CONDA_DEPENDENCIES=""
-               INSTALL_WITH_PIP=True
-               EXTRAS_INSTALL="test,all"
 
-        - os: linux
+        # For the Numpy 1.16 build we use oldestdeps not numpy116 since we can check
+        # the oldest version of all dependencies where this is known.
+        - language: python
+          python: 3.6
+          env: TOXENV="test-oldestdeps"
+               TOXPOSARGS="--open-files"
+
+        # Now try with all optional dependencies.
+
+        - language: python
+          python: 3.6
           stage: Initial tests
-          env: PYTHON_VERSION=3.7 CONDA_DEPENDENCIES=$CONDA_ALL_DEPENDENCIES
-               PIP_DEPENDENCIES="scikit-image $ASDF_PIP_DEP"
-               SETUP_CMD='test -a "--durations=50"'
+          env: TOXENV="test-alldeps"
+               TOXPOSARGS="--durations=50""
           compiler: clang
 
         # Full tests with coverage checks.
-        - os: linux
-          env: SETUP_CMD='test --coverage --remote-data=astropy --readonly'
-               CONDA_DEPENDENCIES=$CONDA_ALL_DEPENDENCIES
-               PIP_DEPENDENCIES="scikit-image codecov objgraph jplephem bintrees sortedcontainers $ASDF_PIP_DEP"
+        - language: python
+          python: 3.7
+          env: TOXENV="test-alldeps"
+               TOXPOSARGS="--remote-data=astropy"
+               COVERAGE=True
                LC_CTYPE=C.ascii LC_ALL=C
                CFLAGS='--coverage -fno-inline-functions -O0'
-               MATPLOTLIB_VERSION=2.1
                EVENT_TYPE='push pull_request cron'
 
         # Try on Windows
         - os: windows
           stage: Final tests
-          env: PYTHON_VERSION=3.7 SETUP_CMD='test --readonly'
```

Comment:
`readonly` is gone? It is the default with `tox`?

Reply:
See #9852